cloud/index.rst

```diff
@@ -23,12 +23,125 @@ The key items you'll care about for each node:
 - Memory - The memory on each node varies based on the size of node you select
 - Cores - The cores on each node varies based on the size of node you select
 
-High Availability
-~~~~~~~~~~~~~~~~~
+Connecting to Citus Cloud
+#########################
 
-The high availability option on a cluster automatically provisions instance stand-bys. These stand-bys receive streaming updates directly from each of the leader nodes. We continuously monitor the leader nodes to ensure they're available and healthy. In the event of a failure we automatically switch to the stand-bys.
+Applications connect to Citus the same way they would PostgreSQL, using a `connection URI <https://www.postgresql.org/docs/current/static/libpq-connect.html#AEN45571>`_. This is a string which includes network and authentication information, and has the form:
 
-Note that your data is replicated to S3 with and without enabling high availability. This allows disaster recovery and reduces the risk of data loss. Although the data is safe either way, we suggest enabling high availability if you cannot tolerate up to one hour of downtime in the rare occurrence of an instance failure. We do not offer a SLA on uptime.
+::
+
+  postgresql://[user[:password]@][host][:port][/dbname][?param1=value1&...]
+
+The connection string for each Cloud Formation is provided on the Overview tab in Citus Console.
+
+.. image:: ../images/cloud-overview-1.png
```

Comment:
Oh, just the creation date and time. Didn't want to date the docs, wanted to keep them feeling fresh in case this section stays around unmodified for some time.